code/__DEFINES/movement_info.dm

```diff
@@ -0,0 +1,21 @@
+#define ACTIVE_MOVEMENT_OLDLOC 1
```

Comment:
Document these, please

Reply:
Whoops, don't need these in this PR. Removed.